monitoring/prometheus.yaml.template

```diff
@@ -0,0 +1,11 @@
+global:
+  scrape_interval:     5s
+  evaluation_interval: 5s
```

Comment:
Let interval be 15 seconds, it's better and it's the value by default. 
I used 5 seconds only for development to not wait too much